tests/unit/src/test/scala/tests/ScalaCliSuite.scala

```diff
@@ -1,5 +1,51 @@
 package tests
 
+import scala.concurrent.Future
+
+import scala.meta.internal.metals.DebugUnresolvedMainClassParams
+import scala.meta.internal.metals.debug.TestDebugger
 import scala.meta.internal.metals.{BuildInfo => V}
 
-class ScalaCliSuite extends tests.BaseScalaCliSuite(V.scala213)
+class ScalaCliSuite extends tests.BaseScalaCliSuite(V.scala213) {
```

Comment:
@tgodzik why `tests.BaseScalaCliSuite` contains tests on its own? Each class extending it will them run over and over. Is this intended?

Reply:
that's now changed